### ISSUE TITLE
Mautic console update 3.0.1 -> 3.0.2 fail with status code 1

### ISSUE THREAD

**Issue by CyberMarsl** [mautic/mautic#9048]
## Mautic update on console (shell) fails

On 3.0.1 installation, the "php bin/console mautic:update:find" finds the 3.0.2
but the command " php bin/console mautic:update:apply" leads instantly to the message "Update abgebrochen."
(it´s german for 'update aborted').

The mautic log warns: mautic.WARNING: Command `mautic:update:apply` exited with status code 1 [] []

| Mautic version | 3.0.1
| PHP version | 7.3.2
Command on terminal as root

### Steps to reproduce
1. Install Mautic 3.0.1
2. In mautic directory at the console hit "php bin/console mautic:update:apply"
 
### Log errors
mautic.WARNING: Command `mautic:update:apply` exited with status code 1 [] []

**Comment by RCheesley** [comment]
Hi @CyberMarsl I just did this update myself without any problem - can you please try again? It might be that there was a temporary glitch in getting the package from Github to do the update.

**Comment by CyberMarsl** [comment]
Well, this time it seemed to work, but: .

1. Somehow it stops on step 5 (clearing cache)
2. installation finisches after hitting:  php bin/console mautic:update:apply --finish

-> only got a white screen on call of mautic page.

3.  I tried to clear the cache manually (remove and reset cache dir permissions)

-> Then it it accessible and works.
--------------------------------------------------------------------------------
I´ve seen 2 other issues in the german version: 

It displays the message "Sind Sie sicher, dass Sie Mautic  zur neuesten Version aktualisieren möchten?" where it has 2 spaces before "Mautic".

Another issue is, that in german you answer with "Ja" or "Nein" as yes or no. So if I hit "j" it aborts. If I hit the english "y" it works.

I tried to look in transifex, but there it seems to be correct without the second space (in all three german languages).

So this update can be closed.
